features/consent/src/main/java/in/projecteka/jataayu/consent/viewmodel/EditConsentDetailsVM.kt

```diff
@@ -140,6 +140,8 @@ class EditConsentDetailsVM(private val repository: ConsentRepository) : ViewMode
             set(Calendar.MINUTE, timePair.second)
             modifiedConsent.permission.dataEraseAt = time.toUtc()
```

Comment:
you can instead write this as 
```
time.toUtc().let {
modifiedConsent.permission.dataEraseAt = it
expiryTimeLabel.set(modifiedConsent.getConsentExpiryTime())
saveEnabled.set(!DateTimeUtils.isDateExpired(it))}
```
this is to avoid one more fetch to the same date.

Reply:
Sure will change